service/cloudconfig/service.go

```diff
@@ -86,6 +88,24 @@ func New(config Config) (*CloudConfig, error) {
 // NewMasterCloudConfig generates a new master cloudconfig and returns it as a
 // base64 encoded string.
 func (c CloudConfig) NewMasterCloudConfig(customObject providerv1alpha1.AzureConfig) (string, error) {
+	// TODO this should be sorted in https://github.com/giantswarm/giantswarm/issues/2383
+	{
+		split := strings.Split(customObject.Spec.Azure.VirtualNetwork.CalicoSubnetCIDR, "/")
+		if len(split) != 2 {
+			return "", microerror.Maskf(invalidCustomObjectError, "Spec.Azure.VirtualNetwork.CalicoSubnetCIDR has invalid fromat %q: it must contain /", customObject.Spec.Azure.VirtualNetwork.CalicoSubnetCIDR)
```

Comment:
nitpick: fromat

i'm assuming that it should be standard cidr notation (e.g: x.x.x.x/y), so it might help to have that here